src/js/events/MesosSummaryActions.js

```diff
@@ -17,7 +17,10 @@ var _historyServiceOnline = true;
 function testHistoryServerResponse(response) {
   // If the response is an empty object, that means something is whack
   // Fall back to making requests to Mesos
-  if (response === {}) {
+  // TODO (DCOS-7764): This should be improved to validate against a schema
+  if (response === {} ||
+      Array.isArray(response.frameworks) ||
```

Comment:
`response === {}` will always be `false`. We could test `Object.keys(response).length`, but there's no real need to have the test at all.